invenio_vocabularies/datastreams/writers.py

```diff
@@ -110,3 +111,21 @@ def write(self, stream_entry, *args, **kwargs):
             yaml.safe_dump([stream_entry.entry], file)
 
         return stream_entry
+
+
+class AsyncWriter(BaseWriter):
+    """Writes the entries asynchronously (celery task)."""
+
+    def __init__(self, writer, *args, **kwargs):
+        """Constructor.
+
+        :param writer: writer to use.
+        """
+        self._writer = writer
```

Comment:
Accepts a writer configuration, that will be sent to the `WriterFactory` later on. I'm not convinced about the name of this parameter. Alternatives, which I dont fully like either are:
- `writer_conf`, `writer_config`, `w_conf`...
- `config`, `conf`...

Everything is too generic....